data-prepper-core/src/main/java/org/opensearch/dataprepper/parser/model/DataPrepperConfiguration.java

```diff
@@ -52,6 +55,7 @@ public DataPrepperConfiguration(
             @JsonProperty("metricRegistries") final List<MetricRegistryType> metricRegistries,
             @JsonProperty("authentication") final PluginModel authentication,
             @JsonProperty("metricTags") final Map<String, String> metricTags,
+            @JsonProperty("metricTagFilters") final List<MetricTagFilter> metricTagFilters,
```

Comment:
Please change this to use snake case. We are updating all of these configurations presently to use snake case and will deprecate the camel case. See #2243